tensorflow/tensorflow.bzl

```
@@ -327,6 +328,9 @@ def tf_copts(
         if_mkl(["-DINTEL_MKL=1", "-DEIGEN_USE_VML"]) +
         if_mkl_open_source_only(["-DINTEL_MKL_DNN_ONLY"]) +
         if_mkl_v1_open_source_only(["-DENABLE_MKLDNN_V1"]) +
+        if_mkldnn_threadpool(["-DENABLE_MKLDNN_THREADPOOL"]) +
+        if_mkldnn_threadpool(["-DENABLE_MKLDNN_V1"]) +
+        if_mkldnn_threadpool(["-DINTEL_MKL_DNN_ONLY"]) +
```

This can be merged into three strings in one list.
```suggestion
        if_mkldnn_threadpool([
          "-DENABLE_MKLDNN_THREADPOOL", 
          "-DENABLE_MKLDNN_V1",
          "-DINTEL_MKL_DNN_ONLY"
        ]) +
```
(Please reapply formatter.)